packaging/enable_nm_trace_logging.sh

```diff
@@ -1,5 +1,3 @@
-install -o root -g root -d /etc/sysconfig/network-scripts
-
 cat > /etc/NetworkManager/conf.d/97-trace-logging.conf <<EOF
```

Comment:
This could now just be the config file that is copied instead of a script that creates th econfig file.

Reply:
Good point. Will do that.